packages/office-addin-dev-certs/scripts/install.ps1

```diff
@@ -0,0 +1,19 @@
+if($args.Count -ne 2){
+    throw "Usage install.ps1 <LocalMachine/CurrentUser> <CA certificate path>"

```

Comment:
It would be more readable to have `Usage: ` so colon and space separates it from the rest.

Reply:
Using pipe would be clearer than slash for a choice: `<LocalMachine | CurrentUser>`

Reply:
Using dashes instead of spaces would be clearer that it is one thing: `<CA-certificate-path>`